airqo-platform/AirQo-frontend
# [NetManager] Fix device overview page

#### Summary of Changes (What does this PR do?)
- Fix device overview page

#### Is this change ready to hit production in its current state?

#### Status of maturity (all need to be checked before merging):


#### How should this be manually tested?
* Install requirements `npm install`
* Start the application `npm run stage-mac` (on macOS) or `npm run stage-pc` (Windows platform)
* Navigate to the device overview page. all the graphs and tables should load

#### What are the relevant tickets?
- [ticket_id]()

#### Screenshots (optional)
